JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -228,8 +228,8 @@ public function denormalize($data, $class, $format = null, array $context = [])
         );
 
         foreach ($normalizedData as $attributeName => $attributeValue) {
-            // Ignore JSON-LD special attributes
-            if ('@' === $attributeName[0]) {
+            // Ignore JSON-LD special attributes and null attribute names
+            if ('@' === $attributeName[0] || is_null($attributeName[0])) {
```

Comment:
How the attribute name can be null? (and you actually test if the first character of the attribute name is null). `{null: "foo"}` isn't a valid JSON document. Can you paste the HTTP request you send to reproduce this error?